test/support/test_data.ex

```diff
@@ -0,0 +1,205 @@
+defmodule MultipassEx.TestData do
+  @moduledoc false
+  use ExUnitProperties
+
+  @dialyzer :no_return
+
+  def site_key do
+    gen all site_key <- string(:alphanumeric, min_length: 32) do
```

Comment:
i could be mistaken, but i don't think you actually need the `gen` block here or in a couple other places in the file